packages/discord-request/src/types.ts

```diff
@@ -19,6 +19,15 @@ export interface Attachment {
 	data: Blob;
 }
 
+type JSONValue =
+	| string
+	| number
+	| boolean
+	| { [x: string]: JSONValue }
+	| undefined
+	| null
+	| JSONValue[];
+
```

Comment:
`body?: BodyInit | unknown;` below should probably be `body?: BodyInit | JSONValue` but that breaks a lot of functionality with discord-api-types. I could serialize everything as a string in `discord-api` instead of doing it here, which would solve this issue, but I think it worsens ergonomics

Reply:
Lets do it in a followup